stable/datadog/templates/daemonset.yaml

```diff
@@ -44,7 +48,44 @@ spec:
       {{- if .Values.daemonset.priorityClassName }}
       priorityClassName: {{ .Values.daemonset.priorityClassName }}
       {{- end }}
+      {{- if .Values.systemProbe.enabled }}
+      initContainers:
+      - name: seccomp-setup
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        command:
+        - cp
+        - /etc/config/system-probe-seccomp.json
+        - /host/var/lib/kubelet/seccomp/system-probe
+        volumeMounts:
+        - name: datadog-agent-security
+          mountPath: /etc/config
+        - name: seccomp-root
+          mountPath: /host/var/lib/kubelet/seccomp
+      {{- end }}
       containers:
+      {{- if .Values.systemProbe.enabled }}
+      - name: system-probe
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        securityContext:
+          capabilities:
+            add: ["SYS_ADMIN", "SYS_RESOURCE", "SYS_PTRACE", "NET_ADMIN"]
```

Comment:
Could we use a list from `values` ? in case we ever need to add capabilities

Reply:
I'd rather make it explicit than flexible, this doesn't change much.